src/client/pfs/pfs.proto

```diff
@@ -191,6 +191,9 @@ message BuildCommitRequest {
   string branch = 4;
   repeated Commit provenance = 2;
   Object tree = 3;
+  // ID sets the ID of the created commit. It also allows you to BuildCommits
+  // with parents that don't exist.
```

Comment:
I think this comment may need to be updated

Reply:
Yup it does.